crates/tracing/src/lib.rs

```diff
@@ -63,3 +63,27 @@ pub fn configure_observability(
         log_level_override,
     )
 }
+
+pub(crate) fn get_http_client(otel_config: &OtelConfig) -> anyhow::Result<reqwest_0_11::Client> {
```

Comment:
A comment here would be great on why we use the very specific version of reqwest